src/tokens/token-groups/font-size.json

```diff
@@ -0,0 +1,15 @@
+{
+  "font-size-1": "12px",
+  "font-size-2": "13px",
+  "font-size-3": "14px",
+  "font-size-4": "15px",
+  "font-size-5": "16px",
+  "font-size-6": "17px",
+  "font-size-7": "20px",
+  "font-size-8": "21px",
+  "font-size-9": "24px",
+  "font-size-10": "26px",
+  "font-size-11": "27px",
+  "font-size-12": "28px",
+  "font-size-13": "42px"
```

Comment:
I never really realized how weird our type scale was. Love having this in one, easily scannable place!